samza-core/src/main/java/org/apache/samza/clustermanager/ContainerManager.java

```diff
@@ -110,12 +119,14 @@ boolean handleContainerLaunch(SamzaResourceRequest request, String preferredHost
       ContainerPlacementMetadata actionMetaData = getPlacementActionMetadata(processorId).get();
```

Comment:
Can the hasActiveContainerPlacementAction and the getPlacementActionMetadata be merged?
active action is any action that is failed or succeeded no?